tst/standard/examples.tst

```diff
@@ -226,6 +226,20 @@ Error, the arguments <n> and <k> must be non-negative integers,
 gap> JohnsonDigraph(IsMutableDigraph, 4, 2);
 <mutable digraph with 6 vertices, 24 edges>
 
-#
+# TadPoleDigraph
+gap> TadpoleDigraph(2, 2);
+Error, m needs to greater than 2
+gap> TadpoleDigraph(3, -1);
+Error, n needs to be positive (or 0)
```

Comment:
This error doesn't immediately tell the user which argument needs to be greater than 2. Perhaps the error messages you could be something more like:
* `the first numeric argument <m> must be an integer greater than 2`
and
* `the final argument <n> must be a non-negative integer`